pyodide-test-runner/pyodide_test_runner/decorator.py

```diff
@@ -44,17 +113,15 @@ def __new__(cls, function: Callable | None = None, /, **kwargs):
 
     def __init__(
         self,
-        *,
-        standalone: bool = False,
-        module_scope: bool = False,
+        selenium_fixture_name: str = "selenium",
```

Comment:
Please update the docstring of `__init__` function according to updated arguments